hw/acpi/pcihp.c

```diff
@@ -93,21 +93,19 @@ static void *acpi_set_bsel(PCIBus *bus, void *opaque)
     return bsel_alloc;
 }
 
-static void acpi_set_pci_info(void)
+static void acpi_set_pci_info(AcpiPciHpState *s)
```

Comment:
The comments indicates

`When the structure is initialized through acpi_pcihp_init() call,
it saves the PCI bus, which means there is no need to invoke a
special function later on.`

We are not saving anything here.  Am I missing something?

Reply:
@mcastelino sorry, didn't quite get the question. We saved AcpiPciHpState->root_bus when acpi_pcihp_init(). So here, we can get the root bus from that. Is this what you ask?